lib/tools/adb-emu-commands.js

```diff
@@ -38,9 +38,24 @@ emuMethods.fingerprint = async function (fingerprintId, udid = undefined) {
   if (udid) {
     await this.setDeviceId(udid);
   }
+  this.resetTelnetAuthToken();
   await this.adbExec(['emu', 'finger', 'touch', fingerprintId]);
 };
 
+emuMethods.rotate = async function (udid) {
+  if (!_.isUndefined(udid) && !(await this.isEmulatorConnected(udid)) ) {
```

Comment:
I would sacrifice the possibility of simulator disconnection to the overall test speed. Each terminal command invocation is expensive from performance point of view